packages/wrangler/e2e/deployments.test.ts

```diff
@@ -236,19 +236,45 @@ const checkAssets = async (testCases: AssetTestCase[], deployedUrl: string) => {
 					);
 				}
 			},
-			{ interval: 1_000, timeout: 30_000 }
+			{ interval: 1_000, timeout: 40_000 }
 		);
 	}
 };
 
-describe.each([
```

Comment:
I got so frustrated trying to get these tests to pass that I ended up re-writing everything. Now, I know there's code duplication in here, but hear me out! In their changed form, these tests are:

- sequentially more readable IMHO 🙇 
- more self contained, because no test depends on the state of previous tests (as it was the case). Each test starts with a clean slate, and cleans up after itself
- more extensive, as they test for extra output
- easier to follow and understand IMHO 🙇